scylla/src/frame/value.rs

```diff
@@ -579,28 +579,28 @@ macro_rules! impl_value_list_for_tuple {
     }
 }
 
-impl_value_list_for_tuple!(T0, T1; 0, 1);
-impl_value_list_for_tuple!(T0, T1, T2; 0, 1, 2);
-impl_value_list_for_tuple!(T0, T1, T2, T3; 0, 1, 2, 3);
-impl_value_list_for_tuple!(T0, T1, T2, T3, T4; 0, 1, 2, 3, 4);
-impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5; 0, 1, 2, 3, 4, 5);
-impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6; 0, 1, 2, 3, 4, 5, 6);
-impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7; 0, 1, 2, 3, 4, 5, 6, 7);
-impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8; 0, 1, 2, 3, 4, 5, 6, 7, 8);
+impl_value_list_for_tuple!(T0, T1; 0, 1; 1);
+impl_value_list_for_tuple!(T0, T1, T2; 0, 1, 2; 2);
+impl_value_list_for_tuple!(T0, T1, T2, T3; 0, 1, 2, 3; 3);
+impl_value_list_for_tuple!(T0, T1, T2, T3, T4; 0, 1, 2, 3, 4; 4);
+impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5; 0, 1, 2, 3, 4, 5; 5);
+impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6; 0, 1, 2, 3, 4, 5, 6; 6);
+impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7; 0, 1, 2, 3, 4, 5, 6, 7; 7);
+impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8; 0, 1, 2, 3, 4, 5, 6, 7, 8; 8);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9; 9);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10; 10);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11; 11);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12; 12);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13; 13);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14; 14);
 impl_value_list_for_tuple!(T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15;
-                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
+                           0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15; 15);
```

Comment:
The tuple sizes are off by one (they should be larger by one). For example, the first `impl_value_list_for_tuple` corresponds to tuple of size 2.